internal/helm/repository/chart_repository.go

```diff
@@ -156,6 +156,14 @@ func newChartRepository() *ChartRepository {
 // to be a semver.Constraints compatible string. If version is empty, the latest
 // stable version will be returned and prerelease versions will be ignored.
 func (r *ChartRepository) GetChartVersion(name, ver string) (*repo.ChartVersion, error) {
+	cv, err := r.getChartVersion(name, ver)
+	if err != nil {
+		return nil, &ErrReference{Err: err}
```

Comment:
I think this is not entirely accurate even though it makes it better. `StrategicallyLoadIndex()`might be subject to transient network failure when downloading the index.

Reply:
Ah, great catch @souleb. We should probably move this loading into this method then, and wrap the returned error from the call in `ErrExternal` (or a third type).